SADL/AM-CDM-part.sadl

```diff
@@ -0,0 +1,31 @@
+uri "http://kdl.ge.com/AM-CDM-part" alias AM-CDM-part version "$Revision:$ Last modified on   $Date:$".
+
+import "http://kdl.ge.com/AM-CDM-base".
+
+Part (note "Instances of part designs in the as-built state.") is a MaterialStock,
+	described by name (note 'Human readable name which designates this part.') with a single value of type string,
+	described by processHistory with a single value of type ProcessHistory,
+	described by partDesignSpecification (note "Design information for the finished part.") with a single value of type DesignSpecification,
+	described by partSpecification (note "Part-specific, reusable process parameters associated with printing this part.") with values of type PartSpecificProcessParameters,
```

Comment:
Do you need import "CDM-Build" here in order to use this class defined in Build?

Reply:
Done